language-server/modules/langserver-core/src/test/resources/formatting/expected/expectedNegativeEOF.bal

```diff
@@ -65,8 +65,7 @@ function getData(json payload) returns [string, string]? {
                 }
             }
         } else
-    {
+{}
```

Comment:
Is this formatting correct?

Reply:
No, with this change, "{ }" can use as an empty block statement and it will change the format. I have mentioned this in tooling team too.

Reply:
Can you create an issue to track this?

Reply:
Okay

Reply:
Sure I will